examples/rcbilling-demo/src/pages/paywall/index.tsx

```diff
@@ -88,12 +83,10 @@ const PaywallPage: React.FC<IPaywallPageProps> = ({ purchases }) => {
 
       console.log(`CustomerInfo after purchase: ${customerInfo}`);
 
-      window.location.href = "/success";
+      navigate(`/success/${purchases.getAppUserId()}`);
     } catch (e) {
       if (e instanceof PurchasesError) {
         console.log(`Error performing purchase: ${e}`);
-        // TODO: We should display an error here.
-        window.location.href = "/";
```

Comment:
Wonder if we should show an alert here... But that can come in a different PR

Reply:
yeah – the main reason I removed this for now is since this also fires when the purchase is cancelled, and then redirecting back to the home page feels very jarring. so we'd probably need better error handling overall.